tests/e2e/k8s/util/kind.go

```diff
@@ -470,6 +486,19 @@ func (c *KindCluster) ScaleDeployment(name, namespace string, replicas int32) er
 	return fmt.Errorf("timeout while waiting for deployment scale to update")
 }
 
+// CreateOperator create operator from folder config.
+func (c *KindCluster) CreateOperator() error {
+	if err := c.CreateFromFolder("./../../../config/operator/manager/"); err != nil {
+		return err
+	}
+	return c.CreateFromFolder("./../../../config/operator/rbac/")
+}
+
+// CreateCRDs creates the CRDs from folder config.
+func (c *KindCluster) CreateCRDs() error {
+	return c.CreateFromFolder("./../../../config/operator/crds/")
+}
+
```

Comment:
I'm trying to keep this file to be general kind-utils, and not clusterlink specific.
I would remove those functions, and inline their code in the caller.